examples/client.py

```diff
@@ -72,6 +76,19 @@ def parse_args():
         print('Closed: {}/{} {}'.format(cc.reason.code, cc.reason.name, reason))
 
 
+async def heartbeat(ws, timeout, interval):
```

Comment:
in case someone might copy this, it would be helpful to enumerate raised exceptions  (TooSlowError, ConnectionClosed, etc.)